solar_system.rb

```diff
@@ -0,0 +1,220 @@
+# solar system program contains a SolarSystem class which can be composed of an array of instances of the Planet class
+# user can create new planets and make them a part of any solar system
+# user can interact with planets, requesting information about any of them
+# user can also interact with a solar system as a whole, requesting the distance between two planets as well as their local year based on their year length and the solar system's age
+
+# ==================
+# class and method definitions
+# ==================
+
+# class Planet lets you create a planet with various attributes and access that information individually and all-together
+# part of wave 2
+class Planet
+  attr_accessor :name, :order, :color, :temp, :water, :diameter, :moons, :year_length, :dist_from_sun
```

Comment:
Why use `attr_accessor`.  In general use `attr_reader` unless you specifically **want** to give the user the ability to change an attribute directly.